packages/form-json-schema/src/defs/component.json

```diff
@@ -227,6 +227,21 @@
     "properties": {
       "$id": "#/component/properties",
       "$ref": "properties.json"
+    },
+    "columns": {
+      "$id": "#/component/columns",
+      "$ref": "columns.json"
+    },
+    "columnsExpression": {
```

Comment:
We could consider making these two exclusive, so only one can exist in the schema.

However for `values`, `valuesKey` and `valuesExpression` we don't have such a rule as we anyway pick one and ignore the others.

Reply:
The message is not very clear, I don't know if there's a better way of doing than the way I did

![image](https://github.com/bpmn-io/form-js/assets/1022916/18f0a18f-bf56-4ff2-85ba-13bfd23b7a86)

Reply:
In the element-templates-json-schema we have a mechanism to provide custom error messages: https://github.com/camunda/element-templates-json-schema/blob/f06ab6b42a32904eff97688294bd9ad4a994ffe3/packages/zeebe-element-templates-json-schema/src/error-messages.json#L29

We'd have the infrastructure already in place in form-js; the task for generating the errors is there. 

However, these messages are rather for our Desktop Modeler tooling. Monaco / VSCode supports the `errorMessage` keyword, but not with string literals, that's why we separate them. What you could try out is using the `errorMessage` keyword directly somewhere in the rule with a static string.

Reply:
Done

![image](https://github.com/bpmn-io/form-js/assets/1022916/4e3b8162-f394-475f-9002-ee6332c5f148)

Reply:
Nice one!